gateway/health_check.go

```diff
@@ -34,12 +34,11 @@ func (gw *Gateway) initHealthCheck(ctx context.Context) {
 
 	go func(ctx context.Context) {
 		var n = gw.GetConfig().LivenessCheck.CheckDuration
-
 		if n == 0 {
-			n = 10
+			n = 10 * time.Second
 		}
 
-		ticker := time.NewTicker(time.Second * n)
+		ticker := time.NewTicker(n)
```

Comment:
if `n` is not zero, will the ticker be created with the value of config but in nanoseconds? While checking the documentation https://pkg.go.dev/github.com/tjgq/ticker#New it seems like it receives seconds? 

So, if someone sets `check_duration: 1000000000`  will it be 10 seconds? or 1000000000 seconds?

Reply:
@sredxny Exactly. Unfortunately, `check_duration` is a `time.Duration` type, which means that the user must set a value in nanoseconds. I'd like to modify it to `int` and express it as seconds, but it would probably break backward compatibilities.
If someone sets `check_duration: 1000000000` it will be 1 second. Nanoseconds = 1e-9 seconds.

> While checking the documentation https://pkg.go.dev/github.com/tjgq/ticker#New it seems like it receives seconds?
Note that the documentation is for a different package (with the same name). Here's the right documentation: https://pkg.go.dev/time#NewTicker